types/steam-tradeoffer-manager/lib/classes/TradeOffer.d.ts

```diff
@@ -0,0 +1,230 @@
+import TradeOfferManager = require("../../index");
+import SteamID = require('steamid');
+import CEconItem = require('steamcommunity/classes/CEconItem');
+import type { InventoryCallback, EResultError } from '../../common';
+
+export = TradeOffer
```

Comment:
FYI: build is failing because of a missing semicolon :)

Reply:
Interesting... Not on my machine... :D
Btw why we even need them in JS?  D

Reply:
> Btw why we even need them in JS?

We don't necessarily, it's just styling enforcement. :)

Although there can be specific small issues that arise from not using semicolons :P